### PR TITLE
Poot

### PR DESCRIPTION
**Pull request by erikroyall**
1. Now defaults NODE_ENV to 'development'
2. Make shit readable ugh
3. Make it 4.x compatible
4. Y U USE TABS EUGH

### CODE REVIEW

**Comment by erikroyall** [comment]
This fixes #3

**Comment by benjamingr** [comment]
Have you signed our CLA? 

Can someone get a GH bot to ping people about the CLA? @Zirak ?

File: start [hunk 1]
```diff
@@ -7,21 +7,29 @@ const express = require('express');
...
+			if (process.env.NODE_ENV !== 'production')
 				res.write(err.message + " " + err.stack);
-			}
+			
 			res.status(500).end();
 			return; 
 		}
-		Function('echo', data)((...args) => res.write(...args));	
+		
+		// The following line doesn't work in 4.x LTS
```
Comment (benjamingr) [inline start:25]: We move fast and break things, 4.x is for losers anyway.

File: start [hunk 2]
```diff
@@ -7,21 +7,29 @@ const express = require('express');
...
+	var path = req.path === '/' ? '/index.js' : req.path;
+	var node_env = process.env.NODE_ENV || 'development';
+	
+	if (path.includes("..") || path.includes("~"))
+		return res.status(503).end("Y U HAX >:( ")
+	
 	fs.readFile(__dirname + path, function(err, data) {
 		if(err) {
-			if(process.env.NODE_ENV !== 'production') {
+			if (process.env.NODE_ENV !== 'production')
```
Comment (ssube) [inline start:18]: :-1: introducing ambiguity

**Comment by awalgarg** [comment]
This repo is now obsolete. Development is going on in https://github.com/awalGarg/jshp since force-push was not working on this repo :/ I blame @MadaraUchiha

**Pull request closed without merging**